### DISCUSSION TITLE
[Bug]: On Export, when you submit using the default value of Last Hour, it says No valid time Range detected

### DISCUSSION THREAD

**Discussion by sylv3rblade** [blakeblackshear/frigate#11246] [General Support] [answered]
### Describe the problem you are having

This is a minor UI bug for the dev build 0.14

![Screenshot 2024-05-05 at 10 01 20 PM](https://github.com/blakeblackshear/frigate/assets/467007/e11a33f5-6f71-496b-84d0-5455b82639ea)

Here's a quick preview of the issue

https://github.com/blakeblackshear/frigate/assets/467007/f39288a5-62ef-470e-ad34-9d3572a07c32

If you click a different option then, re click the Last Hour option it works, it's just the initial load with the default value that exhibits this behavior

### Version

0.14.0-acf37f99

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
fixed in https://github.com/blakeblackshear/frigate/pull/11263
